qiime/core/tests/test_archiver.py

```diff
@@ -125,3 +124,21 @@ def test_save_metadata_file_encoding(self):
                 with io.TextIOWrapper(bytes_fh, newline=None,
                                       encoding='utf-8', errors='strict') as fh:
                     self.assertTrue(fh.read().rstrip('\n'))
+
+    def test_metadata_pprint_yaml(self):
+        archive = self.FakeArchiver()
+        fp = os.path.join(self.root_dir, "archive.qza")
```

Comment:
This should be a temporary file of some kind, unless I'm missing something, this file won't get cleaned up after the tests are finished.